### PR TITLE
fix: explorer input leftovers

### LINKED ISSUE #4930: Explorer input max min left overs
Explorer input max min 
- change the color of icon in dark mode.
-  Let's add a new translation key for it. 
- https://github.com/kodadot/nft-gallery/pull/4924/files/272c608265fc9bddd9f77aa44ac184bd36bce352#diff-243018571f28605fceba0ba9db964d3c1a8b5e9a00152d26f383eb09bd5bd071R39
- https://github.com/kodadot/nft-gallery/pull/4924/files/272c608265fc9bddd9f77aa44ac184bd36bce352#diff-243018571f28605fceba0ba9db964d3c1a8b5e9a00152d26f383eb09bd5bd071R30

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/4924#issuecomment-1419470147_

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4930 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x]  I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Comment by maderi21** [comment]
the `fill="currentColor"` doesn't seem to be working when svg is imported as image. there are also some weird borders around the min/max inputs in dark mode which weren't described in the issue but I can't get rid of them.

regarding payment, same comment as 
- https://github.com/kodadot/nft-gallery/pull/4935#issuecomment-1420666389

**Review by roiLeo — APPROVED** [review]
Noice!
✅ code lgtm & wfm

File: components/explore/SidebarFilter.vue [hunk 1]
```diff
@@ -27,16 +27,27 @@
               type="number"
               min="0"
               step="any"
-              placeholder="MIN"
+              :placeholder="$t('query.priceRange.minPrice')"
               data-cy="input-min" />
-            <img src="/arrow-right.svg" />
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:32]: Let's also delete the svg file.

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
otherwise code lgtm

File: components/explore/SidebarFilter.vue (hunk 1 above)
Comment (maderi21) [reply components/explore/SidebarFilter.vue:32]: arrow svg deleted

**Comment by exezbcz** [comment]
I will create follow up issue for the logic.

**Review by Jarsen136 — APPROVED** [review]

**Comment by vikiival** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 33.37 USD/KSM ~ 0.899 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0x0c27cc6ac4e9b95c6c08f01e3a68c7a097569d231cdface961b440ebe659b7b5](https://kusama.subscan.io/extrinsic/0x0c27cc6ac4e9b95c6c08f01e3a68c7a097569d231cdface961b440ebe659b7b5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**